packages/core/js-client/package.json

```diff
@@ -55,14 +55,13 @@
     "libp2p": "0.46.6",
     "multiformats": "11.0.1",
     "rxjs": "7.5.5",
-    "ts-pattern": "3.3.3",
     "uint8arrays": "4.0.3",
     "uuid": "8.3.2",
     "zod": "3.22.4"
   },
   "devDependencies": {
     "@fluencelabs/aqua-api": "0.9.3",
-    "@fluencelabs/marine-js": "0.7.2",
+    "@fluencelabs/marine-js": "0.8.0",
```

Comment:
Do you need it here? It's already in js-client-isomorphic

Reply:
Yes, it's still used in tests directly from the client.
Will add todo's to remove it.